### DISCUSSION TITLE
Weird mobile error when trying to accept suggestion

### DISCUSSION THREAD

**Discussion by Ethan-Arrowood** [community/community#38233] [Pull Requests]
See screenshot for details-I think it has something to do with the file being an action workflow. I'll have to test at home if I can accept the suggestion through a normal browser

![image](https://user-images.githubusercontent.com/16144158/199864672-389add7a-bb71-41a8-8050-53917a9eedae.png)

**Comment by ossanna16** [comment]
Hi there @Ethan-Arrowood and welcome to our community! Thank you for reporting this 🙂
